docs/developers/tutorial.md

```diff
@@ -747,13 +747,10 @@ custom repository like Bintray or a private Nexus.
 
 ```scala
 // build.sbt
-import com.geirsson.coursiersmall.{Repository => R}
 scalafixResolvers.in(ThisBuild) ++= List(
-  R.SonatypeSnapshots,
-  R.bintrayRepo("scalacenter", "releases"),
-  R.bintrayIvyRepo("sbt", "sbt-plugin-releases"),
-  new R.Maven("https://oss.sonatype.org/content/repositories/snapshots/"),
-  new R.Ivy(s"https://dl.bintray.com/sbt/sbt-plugin-releases/")
+  coursierapi.MavenRepository.of("https://dl.bintray.com/scalacenter/releases"),
```

Comment:
as tested in https://github.com/scalacenter/sbt-scalafix/blob/3c2bbcac8e15b23696daac7d7bf8e8924d6de372/src/sbt-test/sbt-scalafix/local-rules/build.sbt#L12